src/Util/PHP/Template/TestCaseClass.tpl

```diff
@@ -79,9 +90,10 @@ function __phpunit_run_isolated_test()
         }
     }
 
-    file_put_contents(
-        '{processResultFile}',
-        serialize(
+    $fileContents = '';
+
+    foreach ($tests as $test) {
+        $fileContents .= serialize(
```

Comment:
I wonder why/whether it works to concat serialized strings together?

https://3v4l.org/t1ghY